protocol/protocolwrapper/protocol_filter_wrapper.go

```diff
@@ -68,21 +68,16 @@ func (pfw *ProtocolFilterWrapper) Destroy() {
 }
 
 func buildInvokerChain(invoker protocol.Invoker, key string) protocol.Invoker {
-	filtName := invoker.GetUrl().GetParam(key, "")
-	if filtName == "" {
-		return invoker
-	}
-	filtNames := strings.Split(filtName, ",")
-	next := invoker
+	filterName := invoker.GetUrl().GetParam(key, "")
+	filterNames := strings.Split(filterName, ",")
 
 	// The order of filters is from left to right, so loading from right to left
-
-	for i := len(filtNames) - 1; i >= 0; i-- {
-		flt := extension.GetFilter(filtNames[i])
+	next := invoker
+	for i := len(filterNames) - 1; i >= 0; i-- {
```

Comment:
if filterName == "",  
then len(filterNames) == 1 and filterNames[0] == ""。
At this point, the program should return.

Reply:
If `filterName` equals `""`, then `filterNames` will equal to `[]`. Then because `filterNames` is empty string slice, and `len(filterNames)` equals zero, the circle will return immediately.

Reply:
![微信图片_20200902192638](https://user-images.githubusercontent.com/18375310/91975604-3fb2b900-ed52-11ea-84e6-7318ab6d7a8d.png)

@gaoxinge

Reply:
Yes, you are right. I revert this code.